recipes/google-cloud-cpp-core/conda_build_config.yaml

```
@@ -0,0 +1,4 @@
+# The default at [1] is 10.9 and abseil does not compile with that
+# [1]: https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/main/recipe/conda_build_config.yaml
+MACOSX_DEPLOYMENT_TARGET:
+  - 10.13  # [osx and x86_64]
```

Seeing issues with [feedstock creation for this recipe]( https://github.com/conda-forge/admin-requests/actions/runs/7095862148/job/19313442768#step:6:1447 ). Think it has to do with the fact that we are modifying `MACOSX_DEPLOYMENT_TARGET`, but not restricting that modification to `osx and x86_64`, which has an adverse affect on other architectures (like `osx and arm64`). So think we want to add a selector to `MACOSX_DEPLOYMENT_TARGET` (as shown below)

```suggestion
MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
  - 10.13                  # [osx and x86_64]
```

---

Thanks, done in https://github.com/conda-forge/staged-recipes/commit/d4fe07d1ad30b973c5f83b0bb9100c0818bf5a5b

---

Not sure what's happening here. The `create_feedstocks` job still [failed](https://github.com/conda-forge/admin-requests/actions/runs/7095993425/job/19313793818), but an empty https://github.com/conda-forge/google-cloud-cpp-core-feedstock got created already.